build/setup.R

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env Rscript
+
+.args <- commandArgs(trailingOnly = TRUE)
+
+if (length(.args) != 1) {
+  stop("Usage: setup.R <flepimop-path>")
+}
+
+# TODO sniff for upgrade mode
+
+if (!require(devtools)) {
+  install.packages("devtools", repos = c(getOption("repos"), "http://cran.r-project.org"));
+  stopifnot("Could not load `devtools` package." = require(devtools))
+}
+
+rpkgs <- list.files(file.path(.args[1], "flepimop", "R_packages"), full.names = TRUE)
+
+for (pkg in rpkgs) {
+  devtools::install(pkg, reload=FALSE, force=TRUE)
+}
+
+# other dependencies for analysis scripts
+for (pkg in c("ggfortify", "flextable", "optparse", "cowplot")) {
+  if (!requireNamespace(pkg, quietly = TRUE)) { 
+    install.packages(pkg, repos = c(getOption("repos"), "http://cran.r-project.org"));
+    if (!requireNamespace(pkg)) {
+       stop(sprintf("Could not install and/or load `%s` package.", pkg))
+    }
+  }
+}
```

Comment:
these can all be moved into the conda env specification (all just the stand `r-...` names)

Reply:
(agree optparse could go into inference as a depends; makes more sense as a suggests, but there isn't really a way to differentiate "use" vs "develop" suggests, and the "use" environment doesn't need testthat)

Reply:
I think this comment is outdated now. Almost all (minus 1, see https://github.com/HopkinsIDD/flepiMoP/issues/332) of the R dependency management is handled directly through conda. I moved `optparse` into the imports of the `inference` package since that is required to install the CLI tools exposed by that package.